python/enthic/app.py

```diff
@@ -32,6 +32,7 @@
     AverageSirenCompany,
     AllSirenCompany
 )
+from enthic.compute_stats import compute_company_statistics
```

Comment:
This module does not exist in the repository. Perhaps you have just forgot to add it.